wikimedia/wikipedia-ios
# Subscribe and unsubscribe to talk page topics

**Phabricator:** https://phabricator.wikimedia.org/T311639

### Notes
* Add calls to the subscription APIs to get all active subscriptions and subscribe/unsubscribe to a topic
* Update the talk page cells UI accordingly

### Test Steps
1. Go to a talk page, tap the button to subscribe/unsubscribe to a topic
2. Leave the page and when returning, check if your subscriptions are correctly updated
